tests/test_docx.py

```diff
@@ -45,8 +45,6 @@ class ConvertDocxToHtmlTestCase(TestCase):
         # level upperRomans are converted.
         'external_image',
         'export_from_googledocs',
-        'fake_subscript',
```

Comment:
My concern with the fixture approach is that docx fixtures are much more difficult to manage and audit than snippets of XML. As the number of docx fixtures increase, this problem necessarily becomes worse. 

However, I think we definitely want to ensure that if we remove fixture files, we're not losing or missing out on something. I don't think we'll ever remove all of the fixture files, but I feel that reducing or restricting their number, where appropriate, is a good path. To explain why, and hopefully mitigate your concerns, I have the following additional thoughts:

The `DocumentGeneratorTestCase` uses the provided XML to generate valid docx containers as per the specification. That doesn't guarantee that the XML itself is valid, since any arbitrary XML can be used. For that, we continue to use Word/Libreoffice to generate the XML, just like we would for generating a fixture file. However, instead of storing the fixture as a single file, we extract the relevant XML and make it into a test case. That's what I've done in this ticket. I used the relevant XML present in the fixture to form the basis of the new test cases. 

Regarding assumptions, the primary assumption we're working from is that Word produces EMCA-376 valid documents. This assumption isn't arbitrary. That doesn't mean it can't eventually be proven false. However, I don't think one of our docx fixtures will accidentally capture this.